assets.src/src/redmine-tracky/interfaces/time-diff.ts

```diff
@@ -4,20 +4,18 @@ export interface TimeDiff {
     seconds: number;
 }
 
+export function timeDiffToString(timeDiff: TimeDiff) {
+    let res = '';
 
-export function timeDiffToString(timeDiff: TimeDiff): string {
-    const seconds: string = padNumber(Math.round(timeDiff.seconds).toString(), 2);
-    const minutes: string = padNumber(timeDiff.minutes.toString(), 2);
+    const time = [timeDiff.hours, timeDiff.minutes, Math.round(timeDiff.seconds)]
+        .map((value) => value.toString().replace('-', '').padStart(2, '0'))
 
-    if (timeDiff.hours > 0) {
-        return `${timeDiff.hours}:${minutes}:${seconds}`;
-    } else {
-
-        return `${minutes}:${seconds}`;
+    if (timeDiff.minutes < 0 || timeDiff.seconds < 0) {
```

Comment:
* Please use understandable variable names. `sign` might be a good match?

```suggestion
    const  isNegativeDiff = timeDiff.minutes < 0 || timeDiff.seconds < 0);
    const sign = isNegativeDiff ? '-' : '';
```